packages/wallets/src/evm/connectors/smart-wallet/lib/http-rpc-client.ts

```diff
@@ -25,7 +26,11 @@ export class HttpRpcClient {
       {
         url: this.bundlerUrl,
         headers: {
-          "x-api-key": apiKey,
+          ...(clientId
+            ? { "x-client-id": clientId }
+            : secretKey
+            ? { "x-secret-key": secretKey }
+            : {}),
```

Comment:
same here - If `clientId` and `secretKey` both are given - only `x-client-id` header is set